packages/mdc-textfield/foundation.js

```diff
@@ -181,21 +183,20 @@ class MDCTextFieldFoundation extends MDCFoundation {
    * Activates the text field focus state.
    */
   activateFocus() {
-    const {FOCUSED} = MDCTextFieldFoundation.cssClasses;
-    this.adapter_.addClass(FOCUSED);
+    this.isFocused_ = true;
+    this.styleFocused_(this.isFocused_);
     if (this.bottomLine_) {
       this.bottomLine_.activate();
     }
     if (this.outline_) {
       this.updateOutline();
     }
     if (this.label_) {
-      this.label_.floatAbove();
+      this.label_.style(undefined, undefined, undefined, this.isFocused_);
```

Comment:
I find this part so confusing....3 undefined arguments in a row :(

Is there a way to break up the `label.style` method so it doesn't need all 4 arguments?

Reply:
Broke up `style` into `styleFloat` and `styleShake`. Neither method allows `undefined` arguments anymore. I agree I think this is clearer now.